views/ia.erb

```diff
@@ -0,0 +1,32 @@
+<% content_for :head do %>
+	<title>Features - <%= settings.title %></title>
+<% end %>
+
+<% content_for :breadcrumb do %>
+	<%== erb :'partials/_breadcrumb', :locals => {name: 'Features, roadmap and security', href: request.path_info, active: true} %>
+<% end %>
+
+<div class="container">
+	<div class="grid-row">
+
+		<div class="column-one-third">
+			<%== erb :'partials/_subnav', :locals => {active: 'ia'} %>
+		</div>
+
+
+		<div class="column-two-thirds">
+			<h1 class="heading-xlarge">Information assurance</h1>
+
+			<h2 class="heading-medium">Platform assurance</h2>
+			<p>The platform has been assured by the Cabinet Office senior information risk owner (SIRO) and our information and security risk is deemed to be appropriate for data classified as ‘official’. In order to operate with live services, we underwent a formal risk assessment using a methodology based on ISO27005:2011. Our service is currently subject to the Cabinet Office and the Government Digital Service security governance.</p>
```

Comment:
Senior Information Risk Owner is capitalised on line 27, and I think should be here given how we immediately acronym it.